spec.md

```diff
@@ -3724,7 +3800,7 @@ The following example shows the info tokens included:
 
 ``` firrtl
 circuit Top : @[myfile.txt 14:8]
-  module Top : @[myfile.txt 15:2]
+  public module Top : @[myfile.txt 15:2]
     output out: UInt @[myfile.txt 16:3]
     input b: UInt<32> @[myfile.txt 17:3]
     input c: UInt<1> @[myfile.txt 18:3]
```

Comment:
i think we need to add the `public` keyword to the grammar?

And release notes